README.md

```diff
@@ -0,0 +1,19 @@
+# Apache Ambari
+
+Apache Ambari is a tool for provisioning, managing, and monitoring Apache Hadoop clusters. Ambari consists of a set of RESTful APIs and a browser-based management interface.
+
+## Getting Started
+
+https://cwiki.apache.org/confluence/display/AMBARI/Quick+Start+Guide
```

Comment:
later, that would be nice if the README.md itself would contain these descriptions that the wiki have